### ISSUE TITLE
Input in link balloon should have inputmode="url"

### ISSUE THREAD

**Issue by scofalik** [ckeditor/ckeditor5#16389]
## 📝 Provide detailed reproduction steps (if any)

Input in link balloon should have inputmode="url" as it is used to input a URL value.
